app/views/content/landing/is-teaching-right-for-me/_making_a_difference.html.erb

```diff
@@ -0,0 +1,31 @@
+<section class="container">
+  <div>
+    <h2 class="heading--box-blue">Making a difference</h2>
+    <p class="inset">
+      As a teacher, your passion and creativity will inspire and shape future generations.
+    </p>
+  </div>
+</section>
+
+<section class="container centered">
+  <article class="campaign">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "media/images/content/blog/thumbnails/chris-wallis.jpg",
+          heading: "Give something back",
+          text: "I tried many things until I realised that working with young people and helping them develop skills truly inspired me.",
+          accreditation: link_to("Read how Chris changed careers to teaching", page_path("blog/product-engineer-turned-science-teacher")),
+        },
+        {
+          image: "media/images/content/blog/thumbnails/jessica-walmsley.jpg",
+          heading: "Share your subject passion",
+          text: "I love bringing real life science stories into the classroom so students can build knowledge of the world around them.",
+          accreditation: link_to("See how Jessica inspires students with her love of science", page_path("blog/the-head-of-science-inspiring-students-in-blackpool")),
```

Comment:
@ethax-ross is it possible to add a full stop after the link here? (can't quite work out how to do it)